ClangSharp.PInvokeGenerator/PInvokeGenerator.cs

```diff
@@ -164,11 +345,31 @@ private string GetCallingConventionName(Cursor cursor, CXCallingConv callingConv
                     return "Cdecl";
                 }
 
+                case CXCallingConv.CXCallingConv_X86StdCall:
+                {
+                    return "StdCall";
+                }
+
+                case CXCallingConv.CXCallingConv_X86FastCall:
+                {
+                    return "FastCall";
+                }
+
+                case CXCallingConv.CXCallingConv_X86ThisCall:
+                {
+                    return "ThisCall";
+                }
+
+                case CXCallingConv.CXCallingConv_Win64:
+                {
+                    return "Winapi";
+                }
+
                 default:
                 {
-                    Debug.WriteLine($"Unhandled calling convention: {callingConvention} in {cursor.KindSpelling}.");
-                    Debugger.Break();
-                    return string.Empty;
+                    var name = "WinApi";
+                    AddDiagnostic(DiagnosticLevel.Info, $"Unsupported calling convention: '{callingConvention}'. Falling back to '{name}'.", cursor);
```

Comment:
All of the `Debugger.Break`/`Unhandle()` cases were replaced with diagnostics instead.